packages/nimble-components/package.json

```diff
@@ -84,13 +85,13 @@
     "@tiptap/extension-paragraph": "^2.2.2",
     "@tiptap/extension-placeholder": "^2.2.2",
     "@tiptap/extension-text": "^2.2.2",
-    "@tiptap/extension-hard-break": "^2.2.2",
     "@types/d3-array": "^3.0.4",
     "@types/d3-random": "^3.0.1",
     "@types/d3-scale": "^4.0.2",
     "@types/d3-selection": "^3.0.0",
     "@types/d3-zoom": "^3.0.0",
     "@types/markdown-it": "^13.0.0",
+    "apache-arrow": "^15.0.0",
```

Comment:
This should be in peerDependencies since users will be expected to use the library directly and construct objects based on it to pass to us. This will be our first peer dependency.

Reply:
We should probably change [renovate.json](https://github.com/ni/nimble/blob/136a79bc77cf5722f1d06201bd5fa7afe9164bf0/.github/renovate.json#L20) to automatically update `peerDependencies` in addition to the types its already updating.